charts/virtual-kubelet/templates/deployment.yaml

```diff
@@ -26,6 +26,10 @@ spec:
           value: /etc/virtual-kubelet/cert.pem
         - name: APISERVER_KEY_LOCATION
           value: /etc/virtual-kubelet/key.pem
+        - name: LOG_ANALYTICS_ID
+          value: {{ .Values.env.logAnalyticsID }}
+        - name: LOG_ANALYTICS_KEY
+          value: {{ .Values.env.logAnalyticsKey }}
```

Comment:
let's use the LOG_ANALYTICS_AUTH_LOCATION approach, write the ID and key into the vk secret. It will be mounted as a secret volume.